package.ps1

```diff
@@ -23,6 +23,7 @@ Get-ChildItem -Path .\target\* -Include 'azure*' -Exclude '*shaded.jar' | %{ Cop
 StopOnFailedExecution
 copy-item ./worker.config.json pkg
 copy-item ./tools/AzureFunctionsJavaWorker.nuspec pkg/
+Copy-Item ".\lib_worker_1.6.2" pkg\lib -Recurse
```

Comment:
According to the `pom.xml` version is 1.7.1, not 1.6.2. If the number `1.6.2` will be changed in the future, we might better consider it in one place. I can see the several 1.6.2 on the pipeline.

Reply:
The reason for 1.6.2 this is the jars for the latest release we currently have.

Reply:
Sure!